Orm/Xtensive.Orm/Orm/Providers/CommandProcessing/SimpleCommandProcessor.cs

```diff
@@ -51,8 +51,8 @@ void ISqlTaskProcessor.ProcessTask(SqlPersistTask task, CommandProcessorContext
 
     public override void ExecuteTasks(CommandProcessorContext context)
     {
-      context.ProcessingTasks = new Queue<SqlTask>(tasks);
-      tasks.Clear();
+      context.ProcessingTasks = tasks;
+      tasks = new Queue<SqlTask>();
```

Comment:
I'd think about increasing the initial Queue capacity to reduce the reallocation rate.